tests/input/trivial_cycle.xml

```diff
@@ -2,10 +2,10 @@
 
 <simulation>
   <control>
-    <duration>100</duration>
+    <duration>50</duration>
     <startmonth>1</startmonth>
     <startyear>2000</startyear>
-  </control>
+  <solver><config><coin-or><timeout>10</timeout></coin-or></config></solver></control>
```

Comment:
I don't feel really strongly, but I would prefer to see versions of this and ``source_to_sink.xml`` for both greedy and coin solvers - just for completeness.